Xamarin.Forms.Platform.Android/AppCompat/MasterDetailContainer.cs

```diff
@@ -57,6 +57,8 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 			}
 		}
 
+		public void UpdateFlowDirection() => _pageContainer.UpdateFlowDirection(_parent);
```

Comment:
Would we want to maybe do `?.` ?
I see in the method below we set the `_pageContainer = null`. Is there ever a case where this will be null when we call this?